pkg/kubeadm/master/manifests.go

```diff
@@ -213,6 +214,7 @@ func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command
 
 	if component == controllerManager && s.InitFlags.CloudProvider != "" {
 		command = append(command, "--cloud-provider="+s.InitFlags.CloudProvider)
+		command = append(command, DefaultCloudConfig)
```

Comment:
I believe it'd have to check if the file is presenet, and set this flag only if it is present. Also, we would have mount it as a volume, as only `/etc/kubernetes/pki` is bind-mounted to `kube-apiserver` and `kube-controller-manager` right now.